metadata_layer.py

```diff
@@ -807,7 +811,11 @@ def apply(self, edition, metadata_client=None, replace=None,
             replace_rights=False,
```

Comment:
Yes, they should be, but I don't want to take them out until I have a chance to change all the calls to Metadata.apply() in circulation, metadata and content.